UnitySetup/UnitySetup.psm1

```diff
@@ -489,9 +489,10 @@ function Install-UnitySetupInstance {
                 }
                 else { 
                     Write-Verbose "$(Get-Date): Succeeded."
+                    [Environment]::SetEnvironmentVariable("UNITY_$($i.Version)", $Destination, [System.EnvironmentVariableTarget]::User)
```

Comment:
Actually, `$i.Version` is empty string at this point since we're in a different loop from above. You'll need to cache that string for this install process later. Are you having issues testing locally to verify this code works?

In my macOS PR this entire flow is changing and should be easier to get the correct version for this.